src/DotNet.Status.Web/.config/settings.json

```diff
@@ -172,10 +172,6 @@
   "AzureTableTokenStore": {
     "TableUri": "[vault(token-table-sas-uri)]"
   },
-  "DataProtection": {
-    "StorageAccountConnectionString": "[vault(dotnet-status-storage-account)]",
```

Comment:
Does this mean we can get rid of this secret?

Reply:
yes, we wouldn't need this secret with this approach, since we'd be authenticating as a Managed Identity